pkg/cmd/marathon/app/app_list.go

```diff
@@ -1,25 +1,211 @@
 package app
 
 import (
+	"encoding/json"
+	"fmt"
+	"sort"
+	"strings"
+
 	"github.com/dcos/dcos-cli/api"
-	"github.com/dcos/dcos-core-cli/pkg/cmd/marathon/python"
+	"github.com/dcos/dcos-cli/pkg/cli"
+	"github.com/dcos/dcos-core-cli/pkg/marathon"
+	"github.com/dustin/go-humanize"
+	m "github.com/gambol99/go-marathon"
+	"github.com/olekukonko/tablewriter"
 	"github.com/spf13/cobra"
 )
 
+const defaultTableValue = "---"
+
+// `truncate` will break if this is less than 3
+const maxTableValueLength = 30
+
+var deploymentDisplay = map[string]string{
+	"ResolveArtifacts":   "artifacts",
+	"ScaleApplication":   "scale",
+	"StartApplication":   "start",
+	"StopApplication":    "stop",
+	"RestartApplication": "restart",
+	"ScalePod":           "scale",
+	"StopPod":            "stop",
+	"RestartPod":         "restart",
+	"KillAllOldTasksOf":  "kill-tasks",
+}
+
 func newCmdMarathonAppList(ctx api.Context) *cobra.Command {
-	var json bool
+	var jsonOutput bool
 	var quiet bool
 
 	cmd := &cobra.Command{
 		Use:   "list",
 		Short: "List the installed applications.",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return python.InvokePythonCLI(ctx)
+			client, err := marathon.NewClient(ctx)
```

Comment:
Let's follow the pattern used in e.g. https://github.com/dcos/dcos-core-cli/blob/1e66c030d12f83f4658f0cc4155750197563ad24/pkg/cmd/marathon/marathon_about.go#L16 so the actual func is testable and then please add tests.